bin/fix-cocoapods.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env zsh
+set -e
+
+# Installs the CocoaPods dependencies and updates the Gradle cache to recognize that the
+# dependencies are installed.
```

Comment:
My personal copy of this script also has `git restore iosApp/iosApp.xcworkspace/xcshareddata/swiftpm/Package.resolved`, but I'm not sure how often that's actually needed when running this script, and I remember needing it when just switching Git branches and not messing with CocoaPods at all, although now that I'm thinking more about it I'm not sure that I've run into that issue since updating to Xcode 15 (although I run into other issues often enough that I still wouldn't recommend it).